assets/models/system/distilbert-base-cased-distilled-squad/description.md

```diff
@@ -41,10 +41,16 @@ Question Answering | Extractive Q&A | <a href="https://huggingface.co/datasets/s
 ```json
 [
     {
-        "0": "John"
+        "score": 0.9982209801673889,
```

Comment:
The output for above input is: 
```[{"0": "John"}, {"0": "Hyderabad"}]```

(run)[https://ml.azure.com/runs/good_rod_17wb811jx0?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-kartik&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#]

Could you please update?

Reply:
Updated, Thanks